lux/extensions/slack_notifier/slacknotifier.py

```diff
@@ -0,0 +1,71 @@
+import lux
+import json
+import logging
+import pycurl
+from collections import OrderedDict
+
+
+class SlackHandler(logging.StreamHandler):
+    
+    '''Custom handler inheriting from StreamHandler to retain the
+    formatter functionality.
+    Overriding the emit function to use pycurl to send messages to slack
+    (as it turned out to be the most robust option).
+    '''
+    c = pycurl.Curl()
+    c.setopt(c.URL, "https://hooks.slack.com/services/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")
```

Comment:
The whole URL should be taken from settings, or at least should be build from values taken from config (with comments explaining what they are)